IdentityServer/Quickstart/Consent/ConsentController.cs

```diff
@@ -60,9 +60,9 @@ public class ConsentController : Controller
         }
 
         /// <summary>
-        ///     Shows the consent screen
+        /// Indexes the specified return URL.
         /// </summary>
-        /// <param name="returnUrl"></param>
+        /// <param name="returnUrl">The return URL.</param>
         /// <returns></returns>
```

Comment:
Empty returns tag (while this method does return something).
Please, also fix the other empty and missing returns tags in this file :)